### ISSUE TITLE
PHP error because use reserved word "new" in Datatables.php

### ISSUE THREAD

**Issue by plonknimbuzz** [nacasha/CIgniter-Datatables#9]
HI nacasha.

i read your new code , and i found this bug. 
https://github.com/nacasha/CIgniter-Datatables/blob/a83eeabcfd28fe99f45c942b0795736f9b7c540d/libraries/Datatables.php#L20

we can use another word, like `build`

Datatables.php
```
public function build()
	{
		return new DatatablesBuilder();
	}
```
and change in documentation.
README.md
https://github.com/nacasha/CIgniter-Datatables/blob/master/README.md#basic-example
https://github.com/nacasha/CIgniter-Datatables/blob/master/README.md#usage

**Comment by plonknimbuzz** [comment]
reference: https://stackoverflow.com/questions/9575590/why-am-i-getting-an-unexpected-t-new-error-in-php

**Comment by nacasha** [comment]
You really help me a lot finding those bugs :laughing: 
Thank you for your contributions to this repo :smile:

**Comment by nacasha** [comment]
Fixed 870b1caadbf9a2756b513c1e58fe5f153086b399
